endpointmanager/pkg/chplmapper/hackmatch.go

```diff
@@ -0,0 +1,35 @@
+package chplmapper
+
+// this file is for any methods of matching an endpoint to a vendor that is not through the
+// publisher field on the capability statement. These methods should only be used if an endpoint
+// cannot be matched using publisher field on the capability statement.
+
+import (
+	"strings"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/capabilityparser"
+	"github.com/pkg/errors"
+)
+
+func hackMatch(capStat capabilityparser.CapabilityStatement, vendorsNorm []string, vendorsRaw []string) (string, error) {
+	return hackMatchEpic(capStat, vendorsNorm, vendorsRaw)
+}
+
+func hackMatchEpic(capStat capabilityparser.CapabilityStatement, vendorsNorm []string, vendorsRaw []string) (string, error) {
```

Comment:
I think this could be generalized to not be a function that looks for 1 particular vendor. Also I think that doing "hack matches" to create vendor links without somehow noting that we were not able use the field intended to provide us the link (the publisher field) creates a conflict surrounding lantern's ability to give insight into _how_ the FHIR standard is being implemented.

Reply:
> Also I think that doing "hack matches" to create vendor links without somehow noting that we were not able use the field intended to provide us the link (the publisher field) creates a conflict surrounding lantern's ability to give insight into how the FHIR standard is being implemented.

Agree - I didn't want to create the infrastructure in this PR to facilitate capturing these types of discrepancies but fully agree that it should be done.

Reply:
Also, not sure I want to generalize this particular one. I feel like I want the "hack matches" to be super intentional/specific. Maybe we can revisit after we start getting lists with other vendors in them.